.github/workflows/nightly-build-publish.yml

```diff
@@ -59,10 +59,10 @@ jobs:
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
 
       - name: Build and push base images
-        run: /bin/bash arbitrary-users-patch/build_images.sh --push
+        run: /bin/bash -xe arbitrary-users-patch/build_images.sh --push
 
       - name: Build and push happy path image
-        run: /bin/bash arbitrary-users-patch/happy-path/build_happy_path_image.sh --push
+        run: /bin/bash -xe arbitrary-users-patch/happy-path/build_happy_path_image.sh --push
```

Comment:
Also why do we need to use Bash prefix and not ./arbitrary-users-patch/happy-path/build_happy_path_image.sh

Reply:
no idea... that's how it was. But I agree, that's weird.